project_task_project_required/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    is_project_task_project_required = fields.Boolean(
+        string='Require Projects on Tasks',
+        related='company_id.is_project_task_project_required',
+        readonly=False,
```

Comment:
Isn't this the default for related fields?

Reply:
Sinve v12 default is `readonly=True`, see https://github.com/odoo/odoo/blob/897feb553e6ef75af9434cb77436b26abde3a761/odoo/fields.py#L427

Reply:
I see, thanks :+1: